lib/internal/test_runner/test.js

```diff
@@ -710,6 +712,13 @@ class Test extends AsyncResource {
     for (let i = 0; i < this.diagnostics.length; i++) {
       this.reporter.diagnostic(this.nesting, kFilename, this.diagnostics[i]);
     }
+
+    if (this.emitOnlyWarning) {
+      const warning =
+        "to use 'only' or 'runOnly' in the test runner" +
+        ' run node with the --test-only command-line option.';
```

Comment:
`'only' and 'runOnly' require the --test-only command-line option.`

Reply:
Thanks. Updated https://github.com/nodejs/node/commit/56692925d354e6f24f0671f348baf44ae3c10585.